File: 02 - Cpp/01 - Retangulo.cpp
#include <iostream>
#include <iomanip>
#include <cmath>

using namespace std;

main()
{
    double base, altura, perimetro, area, diagonal;

    cout << "Base do retangulo: ";
    cin >> base;
    cout << "Altura do retangulo: ";
    cin >> altura;

    area = base * altura;
    perimetro = 2 * (base + altura);
    diagonal = sqrt(pow(base,2.0) + pow(altura,2.0));

    cout << fixed << setprecision(4);
    cout << "AREA = " << area << '\n';
    cout << "PERIMETRO = " << perimetro << '\n';
    cout << "DIAGONAL = " << diagonal << '\n';

    return 0;
}

File: 04 - Csharp/01 - Retangulo.cs
using System;
using System.Globalization;

namespace Programa
{
    class program
    {
        static void Main(string[] args)
        {
            CultureInfo CI = CultureInfo.InvariantCulture;

            double base_retangulo, altura, area, perimetro, diagonal;

            Console.Write("Base do retangulo: ");
            base_retangulo = double.Parse(Console.ReadLine(), CI);
            Console.Write("Base do retangulo: ");
            altura = double.Parse(Console.ReadLine(), CI);

            area = base_retangulo * altura;
            perimetro = 2 * (base_retangulo + altura);
            diagonal = Math.Sqrt(Math.Pow(base_retangulo, 2.0) + Math.Pow(altura, 2.0));

            Console.WriteLine("AREA: " + area.ToString("F4", CI));
            Console.WriteLine("PERIMETRO: " + perimetro.ToString("F4", CI));
            Console.WriteLine("DIAGONAL: " + diagonal.ToString("F4", CI));
        }
    }
}
